roles/json-filter/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# Git branches
+json_filter_git_branch: develop
+
+# Paths to services folder
+json_filter_workdir: /opt/json-filter
+
+# systemd unit folder
+json_filter_service_dir: /etc/systemd/system
```

Comment:
Just hardcode this service directory. From my perspective, this will never be changed.

Reply:
ok